src/main/java/com/google/api/gax/grpc/PageImpl.java

```diff
@@ -37,24 +37,22 @@
 import com.google.common.collect.Iterators;
 import com.google.common.util.concurrent.Futures;
 import com.google.common.util.concurrent.UncheckedExecutionException;
-
 import io.grpc.StatusRuntimeException;
-
 import java.util.Iterator;
 import java.util.NoSuchElementException;
 import java.util.concurrent.Future;
 
 class PageImpl<RequestT, ResponseT, ResourceT> implements Page<RequestT, ResponseT, ResourceT> {
 
   private final RequestT request;
-  private final FutureCallable<RequestT, ResponseT> callable;
+  private final UnaryApiCallable<RequestT, ResponseT> callable;
   private final PageStreamingDescriptor<RequestT, ResponseT, ResourceT> pageDescriptor;
   private final CallContext context;
   private ResponseT response;
 
   public PageImpl(
       RequestT request,
-      FutureCallable<RequestT, ResponseT> callable,
+      UnaryApiCallable<RequestT, ResponseT> callable,
       PageStreamingDescriptor<RequestT, ResponseT, ResourceT> pageDescriptor,
       CallContext context) {
```

Comment:
Could we change the order of these arguments while we're changing stuff here? I recommend: callable, pageDescriptor, request, context.